dashboards/Query Analytics/pmm-qan.json

```diff
@@ -486,6 +486,5 @@
     },
     "timezone": "",
     "title": "PMM Query Analytics",
-    "uid": "pmm-qan",
```

Comment:
Looks like we may loose the URLS the users may have bookmarked. These id participate in Grafana's paths to dashboards.

Reply:
Yep, it makes sense. I reverted this change